### PR TITLE
A Random Fact Command

### PR DESCRIPTION
**Pull request by csharpisez**
A random fact command using the randfacts package

### CODE REVIEW

File: bot/cogs/facts.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        self.bot = bot
+        
+    @command()
+    async def randomfact(self, ctx: Context) -> None:
+        """Get A Random fact"""
+        randomfact = randfacts.GetFact
+        await ctx.send(embed=Embed(
+            title="Did You Know?",
+            description=randomfact["text"],
+            color=0x690E8
```
Comment (therealdaneel) [inline bot/cogs/facts.py:21]:
```suggestion
        await ctx.send(embed=Embed(
            title="Did You Know?",
            description=randomfact["text"],
            color=Color.blurple()
        ))
```

File: bot/cogs/facts.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import os 
+import Discord
```
Comment (therealdaneel) [inline bot/cogs/facts.py:2]:
```suggestion
from discord import Embed, Color
```

File: bot/cogs/facts.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+from bot.core.bot import Bot
+
+class Facts(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        
+    @command()
+    async def randomfact(self, ctx: Context) -> None:
+        """Get A Random fact"""
+        randomfact = randfacts.GetFact
```
Comment (therealdaneel) [inline bot/cogs/facts.py:17]:
```suggestion
        randomfact = randfacts.getFact()
```

**Review by ActuallyDaneel — CHANGES_REQUESTED** [review]
This command should be placed in the `Fun` cog.
there are various other flake8 inconsistencies that you should also resolve

**Comment by ItsDrike** [comment]
This is a duplicate command to already existing `fact` command in `bot/cogs/fun.py`:
https://github.com/The-Codin-Hole/HotWired-Bot/blob/f2be6e60742bcb387e80fe40131fc8d0a5f8216a/bot/cogs/fun.py#L73-L83

I'm closing this as duplicate

**Pull request closed without merging**